warehouse/slave.go

```diff
@@ -354,7 +394,18 @@ func processStagingFile(job PayloadT, workerIndex int) (loadFileUploadOutputs []
 	// This creates the file, so on successful creation remove it
 	err = jobRun.downloadStagingFile()
 	if err != nil {
-		return loadFileUploadOutputs, err
+		// If error occurs with the current config and current revision is different from staging revision
+		// We retry with the staging revision config if it is present
+		if job.StagingDestinationRevisionID != job.CurrentDestinationRevisionID && job.StagingDestinationConfig != nil {
+			pkgLogger.Infof("[WH]: Starting processing staging file with revision config for StagingFileID: %d, CurrentDestinationRevisionID: %s, StagingDestinationRevisionID: %s, whIdentifier: %s", job.StagingFileID, job.CurrentDestinationRevisionID, job.StagingDestinationRevisionID, jobRun.whIdentifier)
+			err = jobRun.downloadStagingFile()
+			if err != nil {
+				job.sendDownloadStagingFileFailedStat()
+				return loadFileUploadOutputs, err
```

Comment:
We don't need this anymore right ?

Reply:
Oh sorry. Completely missed removing this.